tests/testdata/default_config.yaml

```diff
@@ -329,6 +329,9 @@ train:
   # Number of samples the model should see before saving a checkpoint.
   # Default: 250000
   save_samples: 250000
+  # Optional path to a yaml file that tracks the progress of multiphase training.
+  # Default: None
+  training_journal:
```

Comment:
Should there be an explicit `None` here?

Reply:
by pydantic

Reply:
true